test/controller.js

```diff
@@ -301,4 +301,76 @@ describe('Integration tests', function () {
         assert.isTrue(res.writeHead.calledWith(123));
 
     });
+    
+    it('should provide a way to delete one response for a given endpoint', function () {
```

Comment:
You are right, I'm in the way to undestand some things in testing node.js with chai and others. Will try to improve my test.